client/ayon_blender/api/render_lib.py

```diff
@@ -347,8 +347,9 @@ def prepare_rendering(asset_group):
 
     # Clear the render filepath, so that the output is handled only by the
     # output node in the compositor.
-    bpy.context.scene.render.filepath = ""
-
+    bpy.context.scene.render.filepath = (
```

Comment:
This BigRoy question is still valid https://github.com/ynput/ayon-blender/pull/17#discussion_r1699958299

Reply:
Indeed, the empty path for `bpy.context.scene.render.filepath` is working for some cases. So guess we need to investigate deadline submission issue(e.g. some permission denied eror etc.)
I get inspiration from this website: https://blender.stackexchange.com/questions/119851/render-stops-after-1-frame-render-error-permission-denied-cannot-save-0001